decoder/main.go

```diff
@@ -399,33 +402,33 @@ func UpdatePokemonBatch(ctx context.Context, db db.DbDetails, scanParameters Sca
 		if err != nil {
 			log.Printf("getOrCreatePokemonRecord: %s", err)
 		} else {
-			pokemon.updateFromMap(ctx, db, mapPokemon.Data, int64(mapPokemon.Cell), username)
+			pokemon.updateFromMap(ctx, db, mapPokemon.Data, int64(mapPokemon.Cell), weatherLookup, mapPokemon.Timestamp, username)
```

Comment:
also here? You do it on line 413, but not here?! :) 

What's the advantage of passing the timestamp at all if it's part of the proto you are passing? :)

Reply:
These two actually do take ms since they are expecting `timestampMs` as params.

These timestamps come from remote server so it should be preferred over using local time, which can be out of sync.

Reply:
I messed up but too lazy to change it now. ;)

Reply:
The timestamp in milliseconds from the remote server (only in a GMO) should be the one which is used to determine whether a change is applied (ie latest wins). You can't divide it into seconds as we will be getting lots of GMOs from prospectively multiple workers on the same location if you have raid workers and scanning workers etc

Reply:
Eventually we should change all s to ms. :)